assets/src/block-editor/helpers/index.js

```diff
@@ -942,19 +942,22 @@ export const isAMPEnabled = () => {
 };
 
 /**
- * Replaces the button to preview the post with one that also previews AMP.
+ * Renders the 'Preview AMP' button in the DOM right after the (non-AMP) 'Preview' button.
  */
-export const replacePreviewButton = () => {
-	const postPreviewButton = document.querySelector( '.editor-post-preview' );
-	if ( ! postPreviewButton ) {
+export const renderPreviewButton = () => {
+	const postPreviewButton = document.querySelector( `.${ POST_PREVIEW_CLASS }` );
+	if ( ! postPreviewButton || document.getElementById( AMP_PREVIEW_BUTTON_WRAPPER_ID ) ) {
 		return;
 	}
 
 	const buttonWrapper = document.createElement( 'div' );
-	postPreviewButton.parentElement.insertBefore( buttonWrapper, postPreviewButton.nextSibling );
+	buttonWrapper.id = AMP_PREVIEW_BUTTON_WRAPPER_ID;
 
 	render(
 		<AMPPreview />,
 		buttonWrapper
 	);
```

Comment:
This renders the button into the DOM:

<img width="436" alt="amp-preview-button" src="https://user-images.githubusercontent.com/4063887/65919303-8315e200-e3a1-11e9-996d-10ba0e5a5fce.png">